dapr-spring-core/src/test/java/io/diagrid/spring/core/kvstore/DaprKeyValueTemplateIT.java

```diff
@@ -103,4 +103,30 @@ public void testUpdateDaprKeyValueTemplate() {
         assertThat(updatedType).isNotNull();
     }
 
+    @Test
+    public void testDestroyDaprKeyValueTemplate() throws Exception {
+        var itemId = 1;
+        var insertedType = keyValueTemplate.insert(new TestType(itemId, "test"));
+        assertThat(insertedType).isNotNull();
+
+        keyValueTemplate.destroy();
```

Comment:
We need to check other implementations to make sure this is the right behaviour?

Reply:
In `Spring Data KeyValue` `destroy()` clears the underlying map. Similarly to previous comment we can add `UnsupportedOperationException` so it is clear that we don't support this destructive operation.